src/main/java/org/spongepowered/api/effect/potion/PotionEffect.java

```diff
@@ -90,22 +92,23 @@ static PotionEffect of(Supplier<? extends PotionEffectType> type, int amplifier,
      * Gets the duration in ticks for which this potion effect
      * will apply for.
      *
-     * @return The duration.
+     * @return The duration in ticks.
```

Comment:
This is a good spot - in API-8 this should really be returning `Ticks`.

Reply:
I haven't worked with the Sponge impl before but I'd be happy to try and take a look into doing that :)

Reply:
I'll make the update myself as I'm going to push to get breaking changes all done today, but thanks for the offer!